core/domain/summary_services.py

```diff
@@ -63,6 +63,16 @@ def get_displayable_exp_summary_dicts_matching_ids(exploration_ids):
     displayable_exp_summaries = []
     exploration_summaries = (
         exp_services.get_exploration_summaries_matching_ids(exploration_ids))
+    displayable_exp_summaries = get_displayable_exp_summary_dicts(
+        exploration_summaries, exploration_ids)
+
+    return displayable_exp_summaries
+
+def get_displayable_exp_summary_dicts(exploration_summaries, exploration_ids):
```

Comment:
Yes, this is reasonable. Could you also please make a small modification to the documentation of get_views_multi() to say that some exploration_ids may be None?